src/Components/NewTodo/NewTodo.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { ShapeNewTodo, ShapeNewTodoDefault } from '../Shapes/ShapeNewTodo';
+
+export class NewTodo extends React.Component {
+  state = {
+    currentUserId: '',
+    currentTodo: '',
+    wrongInput: false,
+    errorMessage: '',
+  }
+
+  onChangeUser = (event) => {
+    this.setState({
+      currentUserId: Number(event.target.value),
+      wrongInput: false,
+    });
+  }
+
+  onChangeNewTodo = (value) => {
+    const task = value.replace(/[^\wа-яА-ЯёË ]+|^[ ]+$/gi, '');
+
+    this.setState({
+      currentTodo: task,
+      wrongInput: false,
+    });
+  }
+
+  onSubmitNewTodo = (event) => {
+    event.preventDefault();
+
+    const { currentUserId, currentTodo } = this.state;
+    const { onAdd, users } = this.props;
+
+    let errorMessage;
+
+    if (!currentUserId
+      && !currentTodo) {
```

Comment:
зачем эти переносы? неудобно читать, переносы только если строка длинее 80 символов
```suggestion
    if (!currentUserId && !currentTodo) {
```